src/Service/ProgrammesService.php

```diff
@@ -126,6 +126,26 @@ public function findByPid(Pid $pid, string $entityType = 'Programme'): ?Programm
         return $this->mapSingleEntity($dbEntity);
     }
 
+    /*
```

Comment:
This is !!! but it's gonna be removed once we move towards integrating these changes, so it's fine for the moment